starboard/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ since the version previous to it.
 
 ## Version 16
 
+## Added new configuration constant `kHasPartialAudioFramesSupport`
+Set this to true if your platform supports partial audio frames.
```

Comment:
nit: Set this to false if your platform doesn't support partial audio frames.

Reply:
Good call, updated.